site/content/en/docs/Reference/Configuration/kubernetes.md

```diff
@@ -55,3 +55,9 @@ For instance, to allow Kubernetes to launch on an unsupported Docker release:
 ```shell
 minikube start --extra-config=kubeadm.ignore-preflight-errors=SystemVerification
 ```
+
```

Comment:
add a heading here
maybe
### enable feature gates....

Reply:
Added Heading for feature gate